kefir-test-utils.test-d.ts

```diff
@@ -19,7 +19,7 @@ expectType<void>(
     clock.runToLast()
   })
 )
-expectType<Event<string, number>>(logItem({type: 'value', value: 'hello'}, false))
+expectType<Event<string, string>>(logItem({type: 'value', value: 'hello'}, false))
```

Comment:
Does this fail? I think this is intended to communicate that the Error type can be anything. Which maybe implies it should be `unknown`?

Reply:
Yes, it fails. You can see the failures on the [PR](https://github.com/kefirjs/kefir-test-utils/pull/361) created by dependabot.